ts/src/okx.ts

```diff
@@ -1623,7 +1623,7 @@ export default class okx extends Exchange {
         //        "msg": ""
         //    }
         //
-        const data = this.safeValue (response, 'data', []);
+        const data = this.safeDict (response, 'data', []);
```

Comment:
@rayBastard the defaultType [] does not match the `safeDict` usage

Reply:
Yeah, that should be `safeList`

Reply:
Update it